editor-integration/emacs/dune.el

```diff
@@ -47,6 +47,13 @@
 (defvar dune-error-face 'dune-error-face
   "Face for errors (e.g. obsolete constructs).")
 
+(defface dune-separator-face
+  '((((background light)) (:foreground "firebrick"))
+    (((background dark)) (:foreground "DarkOrange1")))
+  "Face for various kind of separators such as ':'.")
```

Comment:
How come you aren't adding `:group` to these?

Reply:
Alos, I think that instead of setting a concrete color it would be better to inherit from sort of common face. This works with existing color schemes a lot better. The dune mode doesn't really have many elements to its syntax, we should be able to find something to inherit from.

Reply:
> How come you aren't adding :group to these?

It is not necessary: [Put this customization item in group group. If this keyword is missing from a customization item, it'll be placed in the same group that was last defined (in the current file). ](https://www.gnu.org/software/emacs/manual/html_node/elisp/Common-Keywords.html#Common-Keywords), but I can for clarity and robustness.

Reply:
> inherit from sort of common face.

I tried to look at a face that would make sense for `dune-separator-face` and did not find any.  I used `default`.